relay-dynamic-config/src/feature.rs

```diff
@@ -80,16 +80,6 @@ pub enum Feature {
     /// Serialized as `organizations:continuous-profiling-beta-ingest`.
     #[serde(rename = "organizations:continuous-profiling-beta-ingest")]
     ContinuousProfilingBetaIngest,
-    /// Enables metric extraction from spans for common modules.
-    ///
-    /// Serialized as `projects:span-metrics-extraction`.
-    #[serde(rename = "projects:span-metrics-extraction")]
-    ExtractCommonSpanMetricsFromEvent,
-    /// Enables metric extraction from spans for addon modules.
-    ///
-    /// Serialized as `projects:span-metrics-extraction-addons`.
-    #[serde(rename = "projects:span-metrics-extraction-addons")]
-    ExtractAddonsSpanMetricsFromEvent,
```

Comment:
We will have to keep them as deprecated features around at least for a while to make sure managed Relays don't start doing weird things now.